transcom/mymove
# B 19573 ppm est weight review message change 1

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-19573)

## Summary
When creating a shipment as a service counselor, this field would attempt to validate the input which was decorated with a comma, such as ```12,000```

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Log in as a service counselor, and create a shipment for a move.
2. Under the estimated weight section, enter in a weight which would be given a comma delimited value.
3. Swap a number by highlighting the characters to switch, and overwrite the digit.
4. You should see 'Required' if there is no value after changing the input data.
5. You should see 'Please enter a valid number' if the value in the input field is not a numeric value.

### Frontend

- [] This works in [Supported Browsers and their phone views](https://transcom.github.io/mymove-docs/docs/adrs/Browser-Support/#minimum-browser-requirements) (Chrome, Firefox, Edge).

### Backend



## Screenshots
